articles/data-analytics-using-pandas/index.md

```diff
@@ -0,0 +1,126 @@
+
+Pandas provides a robust collection of functions that make it easy to process and read data. In this tutorial, we are going to explore some useful functions and techniques that are an integral part of a data scientist's toolset. You can install Pandas by using Python's package manager, [pip](https://pip.pypa.io/en/stable/). Enter the following command on the terminal:
+
+```bash
+pip3 install pandas
+```
+
+Alternatively, if you want to install Pandas using a different method, [this](https://pandas.pydata.org/docs/getting_started/install.html) tutorial walks you through the various ways in which you can install Pandas. 
+
+### Analyzing data using Pandas
+
+Now that we have Pandas installed on our system, we can delve into data exploration and analysis. For this, I will be using the "wine dataset". Navigate to [this](https://www.kaggle.com/uciml/red-wine-quality-cortez-et-al-2009) link to download the dataset from [Kaggle](https://www.kaggle.com/). *The "wine" dataset is a beginner-friendly dataset that provides information on various factors that affect the quality of the wine*. It has 12 columns describing different factors such as pH, the acidity of the wine, etc.  I will be using **Jupyter notebooks** to execute Python code in this tutorial. However, you can execute the code in a different text editor or IDE of your choice. Jupyter notebooks make it easier to view and explore the data. 
+
+Create a Jupyter notebook by running the following command on the terminal:
+
+```bash
+jupyter notebook
+```
+
+It will open a browser window and display the Jupyter notebook UI. 
+
+### Code
+
+```python
+# Import necessary Libraries
+import pandas as pd 
+
+# read the data using pandas
+data = pd.read_csv("winequality-red.csv")
+```
+The first line imports the Pandas library and gives it an Alias called `pd`. Henceforth, every time we use `pd`, we will be referring to pandas. the `read_csv` function is used to read a [CSV](https://www.howtogeek.com/348960/what-is-a-csv-file-and-how-do-i-open-it/) (comma separated values) file and stored the contents in a variable called `data`.
+
+Pandas stores the read data in a data structure called a [Data Frame](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html). According to the official documentation, a data frame is a Two-dimensional, size-mutable, potentially heterogeneous tabular data structure that also contains labeled axes (rows and columns). In simple terms, a data frame is like a table that contains named columns and rows of data similar to a table in a database. A data frame is powerful and has a lot of in-built functions that allow us to manipulate data. We are going to look at some of these functions. In the above example, the wine dataset is read and stored in a data frame called data. 
```

Comment:
> According to the official documentation, a data frame is a Two-dimensional, size-mutable, potentially heterogeneous tabular data structure that also contains labeled axes 

An explanation for 
- Size-mutable
- heterogeneous
- tabular
- labelled

would be useful for a beginner. It would help them to easily understand the dataframe